index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio setup and mobile first</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+    <header class="header1">
+        <h3 class="logo"><a href="#">My Logo</a></h3>
+        <nav>
+            <img src="./img/hamburger.png" alt="hamburger_image">
+        </nav>
+    </header>
+
+    <section class="section1">
+
+        <div class="intro">
+
+            <h1>I'm Emeka Glad to see you!</h1>
+
+            <p>I'm a software developer! I can help you build a product, feature or website Look through some of my work
+                and experience! if you like what you see and have a project you need coded, don't hesistate to contact
+                me.
+            </p>
+
+        </div>
+
+        <div class="socialmedia_connect">
+
+            <h5>LET'S CONNECT</h5>
+
+            <ul>
+                <li><a href="#"><img src="./img/angular.jpg" alt="angular_image"></a></li>
+                <li><a href="#"><img src="./img/github.jpg" alt="github_image"></a></li>
+                <li><a href="#"><img src="./img/Linkedin.jpg" alt="Linkedin_image"></a></li>
+                <li><a href="#"><img src="./img/mee.jpg" alt="mee_image"></a></li>
+                <li><a href="#"><img src="./img//twitter.jpg" alt=""></a></li>
+            </ul>
+
+        </div>
+    </section>
+
+</body>
+
```

Comment:
- [Optional] Consider using an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)